presto-mongodb/src/test/java/com/facebook/presto/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -270,4 +270,16 @@ public void testRenameTable()
         assertQuery("SELECT value FROM test_rename.tmp_rename_new_table", "SELECT 1");
         assertUpdate("DROP TABLE test_rename.tmp_rename_new_table");
     }
+
+    @Test
+    public void testDeleteDocuments()
+    {
+        assertUpdate("CREATE TABLE test_delete.tmp_delete_rows (value bigint)");
+        MongoCollection<Document> collection = mongoQueryRunner.getMongoClient().getDatabase("test_delete").getCollection("tmp_delete_rows");
+        collection.insertOne(new Document(ImmutableMap.of("value", 1)));
+        assertQuery("SELECT * from test_delete.tmp_delete_rows", "SELECT 1");
+
+        assertUpdate("DELETE FROM test_delete.tmp_delete_rows WHERE value = 1", 1);
+        assertQuery("SELECT count(*) FROM test_delete.tmp_delete_rows", "SELECT 0");
+    }
```

Comment:
Some suggestions for more tests :
delete all rows from the table
delete multiple rows from a table
delete rows based on a complex expression
delete with a subquery (presumably a negative test since from a quick glance it looks like that should not work)

Reply:
Added the all suggested test cases, Could you please check.